resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -78,19 +77,92 @@ class="tw-absolute tw-left-0 tw-top-36 tw-z-10 tw-w-screen tw-px-16">
                             </filter-show-more>
                         </div>
                     </filter-new-popover-group-controller>
-                    <filter-show-more class="tw-visible tw-pt-4 md:tw-hidden"
-                        :is-extended-open="isExtendedOpen"
-                        :toggle-is-extended-open="toggleIsExtendedOpen">
-                    </filter-show-more>
+                    {{-- Mobile Filter --}}
+                    <filter-disclosure-controller v-slot="fm">
+                        <div class="tw-relative md:tw-hidden">
+                            <div class="tw-flex tw-gap-x-3 tw-overflow-x-auto">
+                                <filter-disclosure-button name="authors" @click="fm.goTo('authors')">
+                                    <filter-new-custom-select-popover-label name="authors"
+                                        :selected-values="query['authors']">
+                                    </filter-new-custom-select-popover-label>
+                                </filter-disclosure-button>
+                            </div>
+                            <div class="tw-min-w-max tw-pt-4">
+                                <button @click="fm.goTo('index')"
+                                    class="tw-w-full tw-border tw-border-gray-300 tw-py-3.5 tw-px-4 tw-text-lg tw-font-bold hover:tw-border-gray-800">
+                                    <div class="tw-flex tw-justify-center">
+                                        <div class="tw-flex tw-items-center tw-pr-4">
+                                            <i class="fa fa-sliders"></i>
+                                        </div>
+                                        <span>všetky filtre</span>
+                                    </div>
+                                </button>
+                            </div>
+                            <filter-disclosure-modal v-if="fm.view !== null" @close="fm.close()">
```

Comment:
Just an FYI, you don't need brackets here
```suggestion
                            <filter-disclosure-modal v-if="fm.view !== null" @close="fm.close">
```